esy.json

```diff
@@ -38,18 +35,6 @@
       ["esy-installer", "reason.install"],
       ["esy-installer", "rtop.install"]
     ],
-    "buildsInSource": "_build",
-    "_dependenciesForNewEsyInstaller": {
-      "@esy-ocaml/substs": "^0.0.1",
-      "@esy-ocaml/esy-installer": "^0.0.0",
-      "@opam/ocamlfind": "",
-      "@opam/menhir": " >= 20170418.0.0",
-      "@opam/utop": " >= 1.17",
-      "@opam/merlin-extend": " >= 0.3",
-      "@opam/result": "*",
-      "@opam/ocaml-migrate-parsetree": "*",
-      "@opam/ocamlbuild": "*",
-      "@opam/dune": "*"
-    }
+    "buildsInSource": "_build"
```

Comment:
If build consist of just `dune` invocation it might be useful to try to remove `buildsInSource` config entirely as `dune` supports out of source builds with esy.